v2_jobs.go

```diff
@@ -9,8 +9,11 @@ import (
 
 const jobsBasePath = "api/v2beta/jobs"
 
+// NOTE The jobsDetailsQuery is only available on development branches currently.
+const jobsDetailsQuery = "?detailed"
+
 type JobsService interface {
-	List(context.Context, string, *JobsListRequest) ([]Job, *Response, error)
+	List(context.Context, string, *JobsListRequest, bool) ([]Job, *Response, error)
```

Comment:
I think that the new option could be part of `JobsListRequest, e.g.:

```diff
type JobsListRequest struct {
	Microservice string `json:"microservice,omitempty"`
	LinkID       string `json:"link_uuid,omitempty"`
	Name         string `json:"name,omitempty"`
+
+	// Detailed request, encoded as a query parameter.
+	Detailed bool `json:"-"`
}
```

It reduces the number of breaking changes that we're making to the API.